base/applications/rapps/include/rosui.h

```diff
@@ -200,10 +193,10 @@ class CUiMeasure
         case Type_FitParent:
             return parent;
         }
-
-        return 0;
+        UNREACHABLE;
```

Comment:
That's indeed unreachable since the `switch` statement covers all four possibilities for type.